guides/development/release.md

```diff
@@ -0,0 +1,20 @@
+# Release Process
+
+To issue a new version of any Trento component (e.g. Web, Agent, Wanda, etc.), a new GitHub release needs to be published in the respective repository.  
+Most of the process is automated via GitHub Actions.
+
+## Overview
+
+These are the main steps, to be repeated for each relevant repository:
+
+- Bump the version by changing the file holding the most recent version number, named `VERSION` and located at the root of the repository.
+- Push this change as a commit on the `main` branch (admins only), or open a pull-request as usual.
+  This will trigger the following automated steps:
+  - update the changelog;
```

Comment:
Do we need the semicolon? 
- update the changelog; --> - update the changelog

Also in all the following bullet points

Reply:
I think it's actually a good practice to put punctuation in bullet list, and I've always seen it done this way: semicolons in all but the last one, which gets a full stop instead.